app/views/govuk_publishing_components/components/docs/step_by_step_nav_related.yml

```diff
@@ -62,3 +62,17 @@ examples:
           tracking_id: 'this-is-the-tracking-id'
         }
       ]
+  always_display_as_a_list:
```

Comment:
Can you also add some screenshots to the PR description, showing examples of using it without this flag (i.e: when it's followed by a step-by-step), and with the flag (when it's used for "Also part of" links)?

Reply:
Hi @vanitabarrett - I've added screenshots, hope this gives a bit more context now.